src/components/common/StepComponent.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div class="stepItem">
+    <svg width="30" height="30">
```

Comment:
what was the reason you went with inline svg as opposed to css?

Reply:
I think it was a bit of a brain fart when I started this issue. It's way simpler if make the circles using CSS. Thanks for the feedback!